diff --git a/node_modules/serverless-export-env/src/lib/collectStackOutputs.js b/node_modules/serverless-export-env/src/lib/collectStackOutputs.js
index 8130470..a739e8a 100644
--- a/node_modules/serverless-export-env/src/lib/collectStackOutputs.js
+++ b/node_modules/serverless-export-env/src/lib/collectStackOutputs.js
@@ -13,7 +13,7 @@ function describeStack(AWS, outputs, nextToken) {
         return describeStack(AWS, outputs, response.NextToken);
       }
     })
-    .return(outputs);
+    .then(() => outputs);
 }
 
 /**
diff --git a/node_modules/serverless-export-env/src/lib/resolveCloudFormationEnvVariables.js b/node_modules/serverless-export-env/src/lib/resolveCloudFormationEnvVariables.js
index b027e06..b26542f 100644
--- a/node_modules/serverless-export-env/src/lib/resolveCloudFormationEnvVariables.js
+++ b/node_modules/serverless-export-env/src/lib/resolveCloudFormationEnvVariables.js
@@ -6,14 +6,14 @@ const BbPromise = require("bluebird"),
 function listExports(AWS, exports, nextToken) {
   exports = exports || [];
   return AWS.request("CloudFormation", "listExports", { NextToken: nextToken })
-    .tap((response) => {
+    .then((response) => {
       exports.push.apply(exports, response.Exports);
       if (response.NextToken) {
         // Query next page
         return listExports(AWS, exports, response.NextToken);
       }
     })
-    .return(exports);
+    .then(() => exports);
 }
 
 function listStackResources(AWS, resources, nextToken) {
@@ -29,7 +29,7 @@ function listStackResources(AWS, resources, nextToken) {
         return listStackResources(AWS, resources, response.NextToken);
       }
     })
-    .return(resources);
+    .then(() => resources);
 }
 
 /**
